openshift/_python36.dc.json

```diff
@@ -210,7 +215,7 @@
                 "command": [
                   "bash",
                   "-c",
-                  "cd $FLYWAY_HOME && ./flyway migrate && ./flyway -url=jdbc:postgresql://$FLYWAY_DB_HOST/mds_test migrate"
+                  "cd $FLYWAY_HOME/scripts && ./run_migrations.sh"
```

Comment:
Not required but since you're only running one command now you could maybe simplify the command to just `$FLYWAY_HOME/run_migrations.sh`. If you need to make reference to a script location within a script you can use `SCRIPTDIR="$(dirname "$0")"`.

Reply:
TIL 🙌